projects/task.go

```diff
@@ -536,6 +536,9 @@ type TaskListRequestFilters struct {
 	// TagIDs is an optional list of tag IDs to filter tasks by tags.
 	TagIDs []int64
 
+	// AssigneeIDs is an optional list of tag IDs to filter tasks by tags.
+	AssigneeIDs []int64
```

Comment:
Assingees is generic and could mean user/company/team assignees, I suggest using `AssigneeUserIDs` as it's mapped to `responsiblePartyIds`.

```suggestion
	// AssigneeIDs is an optional list of tag IDs to filter tasks by tags.
	AssigneeUserIDs []int64
```